pkgs/by-name/ca/calibre/package.nix

```diff
@@ -1,43 +1,47 @@
-{ lib
-, stdenv
-, fetchurl
-, cmake
-, fetchpatch
-, fontconfig
-, hunspell
-, hyphen
-, icu
-, imagemagick
-, libjpeg
-, libmtp
-, libpng
-, libstemmer
-, libuchardet
-, libusb1
-, pkg-config
-, podofo
-, poppler_utils
-, python3Packages
-, qmake
-, qtbase
-, qtwayland
-, speechd-minimal
-, sqlite
-, wrapQtAppsHook
-, xdg-utils
-, wrapGAppsHook3
-, popplerSupport ? true
-, speechSupport ? true
-, unrarSupport ? false
+{
+  lib,
+  stdenv,
+  fetchurl,
+  cmake,
+  fetchpatch,
+  ffmpeg,
+  fontconfig,
+  hunspell,
+  hyphen,
+  icu,
+  imagemagick,
+  libjpeg,
+  libmtp,
+  libpng,
+  libstemmer,
+  libuchardet,
+  libusb1,
+  piper-tts,
+  pkg-config,
+  podofo010,
+  poppler_utils,
+  python3Packages,
+  qt6,
+  speechd-minimal,
+  sqlite,
+  xdg-utils,
+  wrapGAppsHook3,
+  popplerSupport ? true,
+  speechSupport ? true,
+  unrarSupport ? false,
 }:
 
+let
+  podofo = podofo010;
```

Comment:
such overrides recommend to keep in all-packages.nix

https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults